tests/storage/test_extraction.py

```diff
@@ -296,7 +483,7 @@ def _rand_name():
     return "".join(random.choice(string.ascii_lowercase) for _ in range(8))
 
 
-def _create_runpath(ert: LibresFacade):
+def _create_runpath(ert: LibresFacade, iteration: int = 0, run: bool = False):
```

Comment:
`_create_runpath` should just create the runpath. Rename it for the `run` to make sense, or create a separate function called `_evaluate_ensemble`.